### DISCUSSION TITLE
How can I buy CMI if I don’t have PayPal?

### DISCUSSION THREAD

**Discussion by groundbreakingmc** [Zrips/CMI#7826] [Q&A]
Pls help

**Comment by CloudeLecaw** [comment]
you cant and this allready written in the faq.
paypal is the ONLY Methode spigot offers.

![Screenshot_2023-05-09-07-10-36-424_com android chrome-edit](https://user-images.githubusercontent.com/12080017/236999737-d2b36260-69f0-4c14-b7fb-9f073989eff9.jpg)


ticket can be closed
